### PR TITLE
[CS-3584] Update ProfileScreen to show message in mainnet

### PR DESCRIPTION
**Pull request by topwebtek7**
### Description

Update profile screen to show 'Switch to Gnosis Chain to view your profile' message when on mainnet

- [x] Completes #CS-3584

### Checklist

- [x] All UI changes have been tested on a small device

### Screenshots

Screen Shot 2022-04-06 at 20 35 00

### CODE REVIEW

**Comment by danibonilha** [comment]
Can we try to make this more user friendly  ? Maybe a smaller font, kind of centering the text and adding a button that redirects to the settings

**Comment by topwebtek7** [comment]
> 

I've tried to make it consistent with Wallet screen, but agree with adding a button redirects to the settings

Screen Shot 2022-04-07 at 20 42 31

File: cardstack/src/screens/ProfileScreen/ProfileScreen.tsx [hunk 1]
```diff
@@ -36,9 +48,15 @@ const ProfileScreen = () => {
       flex={1}
     >
       <MainHeader title={strings.header.profile} />
-      <Container justifyContent="center" flexGrow={1}>
-        {!primarySafe && isFetching ? (
-          <ActivityIndicator size="large" color="white" />
+      <Container
+        justifyContent="center"
+        flexGrow={1}
+        paddingHorizontal={isMainNet ? 5 : 0}
```
Comment (danibonilha) [inline cardstack/src/screens/ProfileScreen/ProfileScreen.tsx]: I believe isLayer1 covers more cases

**Review by lukemelia — APPROVED** [review]

**Review by danibonilha — APPROVED** [review]
Nice one on adding the navigation to network settings 👏

**Pull request merged**